tutorials/tutorial04_meshing/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/presentation/tutorial04_step08_abtr.md

```diff
@@ -0,0 +1,273 @@
+
+!---
+
+## Delete Dummy Assemblies
+
+!row!
+!col small=12 medium=6 large=8
+
+- [BlockDeletionGenerator.md]
+
+- Remove "dummy" assemblies added for core hex patterning
+
+- Set [!param](/Mesh/BlockDeletionGenerator/new_boundary) to same value as outer boundary in `Mesh/core/external_boundary_name` to update the outer boundary sideset along the location of deleted assemblies
+
+!col small=12 medium=6 large=4
+
+!media tutorial04_meshing/hom_abtr_2d_core.png
+       style=width:100%;display:block;margin-left:auto;margin-right:auto;
+
+!row-end!
+
+!listing reactor_examples/abtr/abtr.i
+         block=Mesh/del_dummy
+         link=False
+
+!---
+
+## Extrude 2D core to 3D
+
+- [AdvancedExtruderGenerator.md]
+
+- Extrude 2D mesh to 3D (in +$z$ direction $(0, 0, 1)$)
+
+- Split into multiple intervals, definite heights and number of layers for each
+- Set top/bottom boundary IDs to be referenced later
+
+- Assign new block IDs to each axial level using [!param](/Mesh/AdvancedExtruderGenerator/subdomain_swaps)
+
+!row!
+!col small=12 medium=6 large=8
+
+!listing reactor_examples/abtr/abtr.i
+         block=Mesh/extrude
+         link=False
+
+!col small=12 medium=6 large=4
+
+!media tutorial04_meshing/hom_abtr_3d_core.png
+       style=width:100%;display:block;margin-left:auto;margin-right:auto;
+
+!row-end!
+
+!---
+
+## Assign Plane-level Reporting IDs
+
+!row!
+!col small=12 medium=6 large=8
+
+- [PlaneIDMeshGenerator.md]
+
+- Assign coordinates demarking axial levels in plane_coordinates. These levels should be consistent with how axial levels were defined in [AdvancedExtruderGenerator.md].
+
+!col small=12 medium=6 large=4
+
+!media tutorial04_meshing/hom_abtr_3d_core_with_plane_id.png
+       style=width:100%;display:block;margin-left:auto;margin-right:auto;
+
+!row-end!
+
+!listing reactor_examples/abtr/abtr.i
+         block=Mesh/plane_id
+         link=False
+
+!---
+
+## (Optional) Rename Outer Boundary Sidesets
+
+!row!
+!col small=12 medium=6 large=8
+
+- [RenameBoundaryGenerator.md]
+
+- Griffin requires the outer boundary sidesets to be defined to apply boundary conditions such as vacuum or reflective boundary conditions
+
+!col small=12 medium=6 large=4
+
+!media tutorial04_meshing/hom_abtr_3d_sidesets.png
+       style=width:100%;display:block;margin-left:auto;margin-right:auto;
+
+!row-end!
+
+!listing reactor_examples/abtr/abtr.i
+         block=Mesh/abtr_mesh
+         link=False
+
+!---
+
+## Use of ABTR Mesh in Downstream Physics Code (Griffin)
+
+The Reactor Module creates meshes containing blocks of elements (identified by block ID), groups of elements with similar extra element integers (identified by different extra element IDs such as `pin_id`, `assembly_id`, `depletion_id`), and groups of curves (2D meshes) or faces (3D meshes) called sidesets (identified by sideset ID). In particular, the blocks and sidesets are used in downstream physics codes to assign materials to mesh elements and to assign boundary conditions.
```

Comment:
@aaronoaks Change "extra element integers" to "reporting IDs"

The Reactor Module creates meshes containing blocks of elements (identified by block ID), groups of elements with similar reporting IDs (identified by different reporting IDs such as `pin_id`, `assembly_id`, `depletion_id`), and groups of curves (2D meshes) or faces (3D meshes) called sidesets (identified by sideset ID). In particular, the blocks and sidesets are used in downstream physics codes to assign materials to mesh elements and to assign boundary conditions.